dashboard/src/components/AppLayout.js

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import ResizeObserver from "react-resize-observer";
+
+const styles = theme => ({
+  root: {
+    flex: 1,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "stretch",
+  },
+
+  topBarContainer: {
+    flex: 0,
+    position: "fixed",
+    top: 0,
+    left: 0,
+    right: 0,
+
+    zIndex: 1,
+
+    "@supports (position: sticky)": {
+      position: "sticky",
+    },
+  },
+
+  topBarObserver: {
+    position: "relative",
+  },
+
+  quickNavContainer: {
+    position: "relative",
+  },
+
+  quickNav: {
+    position: "absolute",
+    flexDirection: "column",
+    alignItems: "stretch",
+    width: 72,
+    display: "none",
+
+    paddingTop: 12,
+
+    [theme.breakpoints.up("md")]: {
+      display: "flex",
+    },
+  },
+
+  alertContainer: {
+    position: "relative",
+  },
+
+  alert: {
+    position: "absolute",
+    left: 0,
+    right: 0,
+    height: 0,
+  },
+
+  contentContainer: {
+    flex: 1,
+    display: "flex",
+  },
+
+  content: {
+    flex: 1,
+    position: "relative",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "stretch",
+
+    marginLeft: "auto",
+    marginRight: "auto",
+
+    maxWidth: 1224,
+
+    paddingLeft: 8,
+    paddingRight: 8,
```

Comment:
Why not use the theme's spacing unit here? It's nice to avoid magic numbers when we can.

Reply:
Between this and removing the padding prop from the `Content` component, we've lost a little in translation. 

Previously we have no padding in the content container on narrower viewports, this allowed lists and other elements to expand to the edge of the screen matching platform conventions.